UnitySDK/Assets/ML-Agents/Scripts/Sensor/CameraSensorComponent.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using UnityEngine;
+
+namespace MLAgents.Sensor
+{
+    class CameraSensorComponent : SensorComponent
+    {
+        public new Camera camera;
```

Comment:
Why the `new` keyword here ?

Reply:
Don't this it's needed here

Reply:
```
Assets/ML-Agents/Scripts/Sensor/CameraSensorComponent.cs(8,23): warning CS0108: `MLAgents.Sensor.CameraSensorComponent.camera' hides inherited member `UnityEngine.Component.camera'. Use the new keyword if hiding was intended
```
Should I just use the base member? Not sure how that's normally used.

Reply:
Make a new one. `cAmErA` ?

Reply:
- `SensingCamera`?
 - `SensorCamera`?
 - `AttachedCamera`?
 - `ObservationCamera`?

Reply:
I like ObservationCamera. @surfnerd any preferences?

This means that I either need to fix up the scenes again or use a `FormerlySerializedAs`, so want to make sure I only do it once.

Reply:
Also the base member is deprecated; I'd rather just leave the `new` here unless you feel strongly.

Reply:
Oh wow, yeah that property was removed from `Component` in 2019.2.  `new` is fine.  I didn't realize there was a camera property.